Makefile.ctr

```diff
@@ -176,15 +176,28 @@ STRIP   := $(PREFIX)strip
 NM      := $(PREFIX)nm
 LD      := $(CXX)
 
-ifneq ($(findstring Linux,$(shell uname -a)),)
-	MAKEROM    = pkg/ctr/tools/makerom-linux
-	BANNERTOOL = pkg/ctr/tools/bannertool-linux
-else ifneq ($(findstring Darwin,$(shell uname -a)),)
-	MAKEROM    = pkg/ctr/tools/makerom-mac
-	BANNERTOOL = pkg/ctr/tools/bannertool-mac
+ifeq ($(strip $(CTRBANNERTOOL)),)
+	ifneq ($(findstring Linux,$(shell uname -a)),)
```

Comment:
I know this was already there but... let's try not to use `uname -a` anymore. If for example the string you're searching happens to be found in the machine's hostname, even with the wrong OS, it would match erroneously.

Reply:
How would you prefer I check the host OS?

Reply:
Just `uname` or `uname -s` should work.

Reply:
I already did, I wasnt sure if you wanted jut the -a or uname as a whole but since uname without the -a dosent have that issue I just did that.